dendrite-sample.monolith.yaml

```diff
@@ -181,6 +181,9 @@ client_api:
   recaptcha_bypass_secret: ""
   recaptcha_siteverify_api: ""
 
+#  auto_join_rooms:
+#    - "#main:matrix.org"
+
```

Comment:
Please add some comments as to what is happening with this config. Also, this should be added to the polylith example config.
It should probably be noted which rooms can be added here, as invite only rooms are probably going to cause issues.